### DISCUSSION TITLE
Homebrew automatic publishing

### DISCUSSION THREAD

**Discussion by krupitskas** [premake/premake-core#1577] [Q&A] [answered]
Hi! I've decided to use a premake for a first time after CMake experience, I'm quite surprised that I can't download it through brew, which is expected step for such CLI software. I've saw 3 issues with brew search and I've saw an answer "download premade binary from here". However I can't see any barriers to setup GitHub actions CI for automatic rolling release publishing.
Is it some some principal why we haven't premake at brew (I mean we have, but it really outdated) or just you don't want to spend time on GitHub actions configuration (Here I glad to help)?

**Comment by samsinsane** [comment, marked as answer]
Premake5 is technically in alpha, and most package repositories have a strict policy against alpha software. This is usually why you'll only find Premake4 in package managers, and these are all done by somebody in the Premake community rather than the Premake team. Perhaps once the next full release of Premake occurs we'll consider making official packages, you're not the first and I doubt you'll be the last person to ask us about this.
